src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs

```diff
@@ -295,7 +296,7 @@ private static CultureInfo GetCultureByName(string name)
         /// </summary>
         public static CultureInfo CreateSpecificCulture(string name)
```

Comment:
We're not doing any argument validation on the input name.  Seems like we should be.  If you pass in null, this method will end up throwing NullReferenceException instead of ArgumentNullException.

@safern, @danmosemsft, can we keep a log somewhere of issues found from doing this annotation effort?  Ideally, categorized. e.g. methods that don't expect null and fail with the wrong exception type if you pass null, methods that should be handling null but aren't correctly, etc.?  I think the results of that will be helpful as we look back on this effort and figure out where to take it next.

Reply:
also - presumably - should we be adding checks as we annote it or leave it and assume in the future more people will opt-into this feature and it just won't compile for them? (no personal preference either way but since we chose compile errors I think it makes sense not to do any checks)

Reply:
> @safern, @danmosemsft, can we keep a log somewhere of issues found from doing this annotation effort? Ideally, categorized. e.g. methods that don't expect null and fail with the wrong exception type if you pass null, methods that should be handling null but aren't correctly, etc.? I think the results of that will be helpful as we look back on this effort and figure out where to take it next.

Would an issue work fine?

Reply:
Created one. https://github.com/dotnet/corefx/issues/36384

Reply:
let's just reference it without any notes on the PRs